README.md

```diff
@@ -9,8 +9,8 @@ screenshot is worth a thousand words:
 Mop is implemented in Go and compiles down to a single executable file.
 
     # Make sure your $GOPATH is set.
-    $ go get github.com/michaeldv/mop
-    $ cd $GOPATH/src/github.com/michaeldv/mop
+    $ go get github.com/mop-tracker/mop
```

Comment:
why not just `go get github.com/mop-tracker/mop/cmd/mop`

Reply:
question... is this still relevant? if so, how? this so I can integrate your suggestion in the script, thank you.